altsrc/flag_generated.go

```diff
@@ -4,343 +4,271 @@ package altsrc
 
 import (
 	"flag"
-	"github.com/urfave/cli"
```

Comment:
❓ _(blocking question)_ this file should be removed, yea?

Reply:
Yes. I think I need to look the altsrc code once again

Reply:
I am going to reorganize the altsrc package for good. You should see something exciting in the next commit :)

Reply:
If you're about to make large changes to `altsrc`, I would specifically advise against doing that inside this PR!

Reply:
I will do a separate PR for that.

Reply:
# 907